source/clear-linux/bundles/bundles.rst

```diff
@@ -1,5 +1,5 @@
-Bundles
-#######
+Reference
```

Comment:
Several things here: This file lacks a starting label .. _reference:
Sometimes, changing the filenames and directories is unavoidable. The reference content about bundles is part of the Reference content section, therefore, changing the heading here is not enough. Here, I would create the reference directory, place a reference.rst file and include the bundles.rst file in that toctree. Otherwise, you create a disparity between the content and the structure.